<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dobble</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="6" colspan="2">
    <widget class="QPushButton" name="buttonH">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QLabel" name="playerName6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="playerName2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="playerName5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="2">
    <widget class="QLabel" name="labelH">
     <property name="text">
      <string>H</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="playerName3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_4">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="labelF">
     <property name="text">
      <string>F</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="17" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_8">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="16" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_7">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="8">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QPushButton" name="buttonF">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="9">
    <widget class="QLabel" name="countDown">
     <property name="minimumSize">
      <size>
       <width>357</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="4">
    <widget class="QLabel" name="playerCount">
     <property name="text">
      <string>Number of players: 0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="buttonE">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QPushButton" name="buttonB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="labelA">
     <property name="text">
      <string>A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLabel" name="labelB">
     <property name="text">
      <string>B</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="QLabel" name="labelD">
     <property name="text">
      <string>D</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_3">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QPushButton" name="buttonC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QLabel" name="labelC">
     <property name="text">
      <string>C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_6">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="15" column="7">
    <widget class="QLabel" name="cardsLeft6">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="7">
    <widget class="QLabel" name="cardsLeft7">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="playerName1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="7">
    <widget class="QLabel" name="cardsLeft3">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="7">
    <widget class="QLabel" name="cardsLeft2">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="labelE">
     <property name="text">
      <string>E</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="playerName4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QLabel" name="playerName7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="6" colspan="2">
    <widget class="QPushButton" name="buttonD">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </item>
   <item row="9" column="4" colspan="2">
    <widget class="QPushButton" name="buttonG">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
   </item>
   <item row="17" column="7">
    <widget class="QLabel" name="cardsLeft8">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_5">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QLabel" name="labelG">
     <property name="text">
      <string>G</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="7">
    <widget class="QLabel" name="cardsLeft5">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QLabel" name="playerName8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="7">
    <widget class="QLabel" name="cardsLeft4">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="buttonA">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="5">
    <widget class="QProgressBar" name="progressBar_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="10" column="7">
    <widget class="QLabel" name="cardsLeft1">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
